src/main.cpp

```diff
@@ -4,23 +4,32 @@
 #include "configInterface.h"
 
 bool dual_tor_sock = false;
+char loopback[IF_NAMESIZE] = "Loopback0";
 
 static void usage()
 {
-    printf("Usage: ./dhcp6relay {-d}\n");
-    printf("\t-d: enable dual tor option\n");
+    printf("Usage: ./dhcp6relay [-u <loopback interface>]\n");
```

Comment:
Why is the change only being made in 202012, is there a plan to update latest branches?

Reply:
Is Loopback0 interface a mandatory configuration during ToR deployments?

Reply:
Yes, I will open another PR for latest branches soon.

For Loopback0 interface, we also need following code change in build-image repo, I am going to open a PR for this.
It's mandatory configuration for dual-tor, but not needed in single tor.

diff --git a/dockers/docker-dhcp-relay/dhcpv6-relay.agents.j2 b/dockers/docker-dhcp-relay/dhcpv6-relay.agents.j2
index 8f83e05efc..e392711f53 100644
--- a/dockers/docker-dhcp-relay/dhcpv6-relay.agents.j2
+++ b/dockers/docker-dhcp-relay/dhcpv6-relay.agents.j2
@@ -13,7 +13,7 @@
 [program:dhcp6relay]
 command=/usr/sbin/dhcp6relay
 {#- Dual ToR Option #}
-{% if 'subtype' in DEVICE_METADATA['localhost'] and DEVICE_METADATA['localhost']['subtype'] == 'DualToR' %} -d{% endif %}
+{% if 'subtype' in DEVICE_METADATA['localhost'] and DEVICE_METADATA['localhost']['subtype'] == 'DualToR' %} -u Loopback0 {% endif %}
 
 priority=3
 autostart=false
diff --git a/src/dhcprelay b/src/dhcprelay
index a4b15d8ba4..41827a7338 160000

Reply:
Okay, Thanks